app/controllers/iiif/auth/v2/probe_service_controller.rb

```diff
@@ -59,8 +60,22 @@ def auth_probe_result(file, cocina)
             else
               AuthProbeResult2.ok
             end
-          else
+          elsif file.stanford_restricted?
             AuthProbeResult2.unauthorized(unauthorized_heading(file))
+          else
+            AuthProbeResult2.forbidden(*forbidden_fields(file))
+          end
+        end
+        # rubocop:enable Metrics/AbcSize, Metrics/PerceivedComplexity
+
+        def forbidden_fields(file)
```

Comment:
Instead of a tuple, can we return a hash with `:note` and `:icon` keys?  I think the tuple makes it a bit more challenging to follow.